contracts/sol6/KyberNetwork.sol

```diff
@@ -1026,9 +1028,9 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
         require(input.src != input.dest, "src = dest");
 
         if (input.src == ETH_TOKEN_ADDRESS) {
-            require(msg.value == input.srcAmount, "bad eth qty");
```

Comment:
why do we remove revert messages here?

Reply:
I think the main reason was that it the proxy will also catch this case...? But it probably will reach this line first.

The other reason is so that we increase the solc runs.

Can put the msg back.

Reply:
Reason is it shouldn't happen.
and if it does it should be caught on proxy level and it will issue a message (we should fix proxy message...)

so enough to have require without message for cases that will revert on proxy level.

But @Anyhowclick why u say it will reach this line first?

Reply:
Proxy doesn't check if srcAmount == msg.value if src == ETH, that's why it will be reverted here in Network first

Reply:
Right
I fixed that now in another PR
we better have it on proxy level.

Reply:
Ok, I'm reverting any changes made to revert msgs.